compiler/src/main/java/com/squareup/anvil/compiler/InterfaceMerger.kt

```diff
@@ -131,32 +129,20 @@ internal class InterfaceMerger(
     val excludedClasses = mergeAnnotation.exclude()
       .filter { it.isInterface() }
       .onEach { excludedClass ->
-        val contributesToAnnotation = excludedClass
-          .annotations.find(contributesToFqName).singleOrNull()
-        val contributesBindingAnnotation = excludedClass
-          .annotations.find(contributesBindingFqName).singleOrNull()
-        val contributesMultibindingAnnotation = excludedClass
-          .annotations.find(contributesMultibindingFqName).singleOrNull()
-        val contributesSubcomponentAnnotation = excludedClass
-          .annotations.find(contributesSubcomponentFqName).singleOrNull()
+        val scopes = excludedClass.annotations.find(contributesToFqName).map { it.scope() } +
+          excludedClass.annotations.find(contributesBindingFqName).map { it.scope() } +
+          excludedClass.annotations.find(contributesMultibindingFqName).map { it.scope() } +
+          excludedClass.annotations.find(contributesSubcomponentFqName).map { it.parentScope() }
```

Comment:
I'll make this pattern a little bit nicer in a follow-up PR by introducing `findAll`. I wanted to make the change separately to keep the size of this PR a little smaller.

Reply:
#552